lua/presence/init.lua

```diff
@@ -466,6 +466,12 @@ function Presence:update_for_buffer(buffer, should_debounce)
     }
 
     local editing_text = self.options.editing_text
+    if vim.bo.modifiable then
```

Comment:
Is this true? It would be trivial to add, but from my own testing it already works perfectly fine with help files.

Reply:
Right, for help files `vim.bo.modifiable` is set to `false` so your change works perfectly for any unmodifiable file whether it's a help file or some file opened using the `-M` option. For read-only or immutable files however, `vim.bo.modifiable` is actually set to `true` since you can still edit the files, but run into warnings when either editing and saving the file:
```sh
$ touch read-only.txt      # Create a new file
$ chmod 0444 read-only.txt # Make it a read-only file
$ nvim read-only.txt       # Discord will show "Editing read-only.txt"
```

I've been internally debating and I can go either way on this since _technically_ you can be editing the file, so I'll resolve this but feel free to let me know what you think!

Reply:
I think that if the majority of time you're reading instead of editing these immutable files, then it should default to "reading". Users can set `vim.bo.modifiable` to true if they want the "editing" text message